packages/backend/convex/env.ts

```diff
@@ -8,6 +8,7 @@ export const env = createEnv({
     POLAR_ACCESS_TOKEN: z.string().min(1),
     POLAR_ORGANIZATION_ID: z.string().min(1),
     POLAR_WEBHOOK_SECRET: z.string().min(1),
+    POLAR_SERVER: z.enum(["sandbox", "production"]).optional(),
```

Comment:
I'd love to see this pattern for preview deploys, OT but would love to hear your take on how to do preview deploys with separate backends AND separate data (where Convex differs from many systems)

Reply:
Yeah it's tricky with integrations like this, I've had similar challenges with Plaid. For those cases I've kept previews on sandbox and created new test accounts to validate PR changes. Would be cool to build up some formality around that flow.